app/views/api/guidance/guidance_for_lead_providers/partnerships.md

```diff
@@ -0,0 +1,156 @@
+# Create, view and update partnerships 
+
+## What are partnerships? 
+
+Partnerships are agreements between schools, lead providers, and delivery partners to deliver training to early career teachers (ECTs) and mentors for a given cohort.  
+
+## Partnerships process overview 
+
+1. Cohort opens for registrations.
+2. Schools register ECTs and mentors, choosing the programme type (`provider-led` or `school-led`) for each participant rather than making a schoolwide choice.
+3. Lead providers find schools they want to partner with using `GET /schools` API endpoint.
+4. Lead providers create partnerships via the `POST /partnerships` endpoint.
+
+## Understanding partnerships in the API 
+
+The API allows lead providers to confirm, view, and update these agreements so that participants are correctly assigned to training. 
+
+Lead providers should always check for existing partnerships before creating a new one: 
+
+* use `GET /partnerships` to see if a partnership already exists for the school and cohort
+* use `GET /schools` to check which programme type the school has chosen and if it’s already partnered with another lead provider 
+
+Submitting a duplicate partnership (same school, lead provider, and cohort) will return a `422 error`. 
+
+Additional partnerships by different lead providers for the same school and cohort are now allowed. This reflects the fact that some schools work with multiple lead providers in a single cohort. 
+
+### Partnership ID 
+
+Each confirmed partnership has a unique identifier (`id`), returned in the `GET /partnerships` response. This ID represents the agreement between a school, delivery partner, and lead provider for a given cohort. 
+
+### Programme choices now set per participant 
+
+We’ve removed challenge fields (`challenged_reason`, `challenged_at`, `status`) from partnership responses because schools now make changes at an individual participant level (for example, moving an ECT to a different provider) rather than challenging a school-wide partnership. 
```

Comment:
To see if a school has any ECTs training with you, we've introduced a new field 'participants_currently_training'. This will show the number of ECTs or mentors training with you currently for that partnership. It will show 0 when:
- no participants are registered for training with you for traiing 
- all participants complete training
- all participants are moved to another lead provider or school-led training

I think we need this somewhere, could add in a later PR tho

Reply:
I think we said about maybe adding this later so happy for that too